config/routes.rb

```diff
@@ -8,6 +8,7 @@
   namespace :api do
     namespace :v1 do
       resources :jobs, only: [:index, :show]
+      resources :companies, only: [:show]
       get '/recent_jobs', to: "recent_jobs#index"
```

Comment:
I'm a fan of being explicit with these routes, like it is on line 12. This isn't a comment about consistency, but just that it's easier for me to quickly tell what's going on with routes if I don't have to remember all the renaming that rails does.